docs/user_deletion.md

```diff
@@ -0,0 +1,24 @@
+# User Deletion
+
+In it's default setup, deleting a user account revokes their access to the application and soft-deletes the User record. We use the [discard](https://github.com/jhawthorn/discard) gem to soft-delete records across the application.
+
+Depending on legalities of your application, you can choose from three options:
+
+1. Soft-delete(default)
+2. Soft-delete and anonymization
+3. Permanentaly delete the user record
+
+## How to delete a user account
+
+1. A signed in user goes to their **Account Settings** and clicks the **Delete Account** button, enters their password and optional feedback and then click the **Delete Account Now** button. This will soft-delete the record and sign them out.
+
+## User Anonymization and Customization
+
+We have an optional rake task that if setup, anonymizes user records after a period of time. This task calls `AnonymizationService` which strips all columns of the user's data.
```

Comment:
link to the file in our GitHub repo so they can click on the AnonymizationService and view the file if they would like to

Reply:
same for other things we mention in the code. like below for the variable that they need to set. we want to make it easy for users to jump to code places without them having to go and have a look at things themselves.